src/yz_fuse.erl

```diff
@@ -124,18 +124,19 @@ fuse_context() ->
 %% bucket in Riak) and must be an atom, such as yz_index for example. 
 -spec encode_fuse_name(Namespace::atom(), Name::binary()) -> encoded_fuse_name().
 encode_fuse_name(Namespace, Name) ->
-    Fuse = {fuse, Namespace, Name},
-    Base64 = base64:encode(term_to_binary(Fuse)),
-    binary_to_atom(Base64, latin1).
+    list_to_atom(atom_to_list(Namespace) ++ ":" ++ binary_to_list(Name)).
 
 %% @doc Given an atom generated by the encode_fuse_name/2 function, decodes it
 %% into a tuple of the form {fuse, Namespace, Name}, where Namespace and Name
 %% are the arguments to encode_fuse_name.
 -spec decode_fuse_name(encoded_fuse_name())
-                      -> {fuse, Namespace::fuse_namespace(), Name::fuse_name()}.
-decode_fuse_name(Name) ->
-    Fuse = base64:decode(atom_to_binary(Name, latin1)),
-    {fuse, _Namespace, _Name} = binary_to_term(Fuse).
+                      -> {Namespace::fuse_namespace(), Name::fuse_name()}.
+decode_fuse_name(Name0) ->
+    Name = atom_to_list(Name0),
+    Index = string:chr(Name, $:),
```

Comment:
If there is no : in the Name it will fail, but that seems to be okay. That shouldn't happen.